package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/withastro/compiler.git"
   },
   "scripts": {
-    "build": "make wasm",
+    "build": "CGO_ENABLED=0 GOOS=js GOARCH=wasm go build \"-ldflags=-s -w\" -trimpath -o ./packages/compiler/wasm/astro.wasm ./cmd/astro-wasm/astro-wasm.go",
```

Comment:
what's this about?

Reply:
So that it isn't necessary to install `make` to contribute to the project, just pnpm and go.